File: app/src/main/java/xyz/lgvalle/chachi/MainActivity.java
package xyz.lgvalle.chachi;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main_activity);

        if (savedInstanceState == null) {
            ArticleListFragment fragment = new ArticleListFragment();

            getSupportFragmentManager().beginTransaction()
                    .add(R.id.fragment_container, fragment, ArticleListFragment.TAG)
                    .addToBackStack(ArticleListFragment.TAG)
                    .commit();
        }
    }
}

File: app/src/main/java/xyz/lgvalle/chachi/MainActivity.kt
package xyz.lgvalle.chachi

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.Menu

class MainActivity : AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.main_activity)

        if (savedInstanceState == null) {
            val fragment = ArticleListFragment()
            supportFragmentManager.beginTransaction()
                    .add(R.id.fragment_container, fragment, ArticleListFragment.TAG)
                    .addToBackStack(ArticleListFragment.TAG)
                    .commit()
        }
    }

    override fun onCreateOptionsMenu(menu: Menu): Boolean {
        menuInflater.inflate(R.menu.main_activity, menu)
        return true
    }


}
